aws/data_source_aws_organization_attributes.go

```diff
@@ -0,0 +1,91 @@
+package aws
```

Comment:
Why not simply naming the file data_source_aws_organization.go? (and other ones below)
Isn't this the main object representing an Organization?